docs/content/what-is-a-run-id.md

```diff
@@ -0,0 +1,21 @@
+---
+id: what-is-a-run-id
+title: What is a Run Id?
+description: A Run Id is a platform-level identifier for a Workflow Execution that is unique within a Namespace.
+tags:
+  - explanation
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+A Run Id is a platform-level identifier for a [Workflow Execution](#) that is unique within a [Namespace](docs/server/namespaces).
+
+Temporal guarantees that only one [Workflow Execution](#) with a given [Workflow Id](what-is-a-workflow-id) can be open at a time.
+But after the [Workflow Execution](#) has completed, if allowed by a configured policy, you might be able to re-execute a [Workflow](#) after it has closed or failed by using the same [Workflow Id](what-is-a-workflow-id).
```

Comment:
Is there any way to add an example or link to an example of how you could configure a policy that allows re-executing a completed workflow?